clients/src/main/resources/common/message/OffsetForLeaderEpochRequest.json

```diff
@@ -32,13 +32,13 @@
     { "name": "Topics", "type": "[]OffsetForLeaderTopic", "versions": "0+",
       "about": "Each topic to get offsets for.", "fields": [
       { "name": "Name", "type": "string", "versions": "0+", "entityType": "topicName",
```

Comment:
Previous "name" is ```topic``` (https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/protocol/CommonFields.java#L25) and the "name" in auto-generated protocol is ```name```. Does it break the compatibility? For example, a cluster mixed by auto-generated protocol and stale protocol.

Reply:
Please ignore above comment. I misunderstood the protocol :(